h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -578,4 +583,20 @@ public void done(Frame frame) {
     }
   }
 
+  static Consumer<Vec> dropit = new Consumer<Vec>() {
+    @Override public void accept(Vec v) { v.remove(new Futures()).blockForPending(); }
+  };
+
+  @BeforeClass
+  public static void hi() { stall_till_cloudsize(1); }
+  @AfterClass public static void bye() { toDrop.forEach(dropit); }
+
+  private static Set<Vec> toDrop = new HashSet<>();
+
+  protected static Vec willDrop(Vec v) { toDrop.add(v); return v; }
+
+  protected static <T extends Vec.VectorHolder> T willDrop(T vh) {
```

Comment:
agree with Michal :+1:

We have Scope for tracking keys to deallocate. We do not need another mechanism for ref tracking.